metricbeat/module/kafka/consumergroup/_meta/fields.yml

```diff
@@ -0,0 +1,62 @@
+- name: consumergroup
+  type: group
+  description: >
+    consumergroup
+  fields:
+    - name: broker
+      type: group
+      description: >
+        Broker Consumer Group Information have been read from (Broker handling
+        the consumer group).
+      fields:
+        - name: id
+          type: long
+          description: >
+            Broker id
+
+        - name: address
+          type: keyword
+          description: >
+            Broker address
+
+    - name: group
```

Comment:
This should be `id`, as it will be `kafka.consumegroup.id`. Alternative is `kafka.conumser.group`. Is the partition, topic, offset specific to a consumergroup or a consumer?